completed/camunda-8.js

```diff
@@ -2,18 +2,19 @@ import axios from "axios";
 import { getAccessToken } from "./auth.js";
 
 const authorizationConfiguration = {
-  clientId: process.env.ZEEBE_CLIENT_ID,
-  clientSecret: process.env.ZEEBE_CLIENT_SECRET,
-  audience: process.env.ZEEBE_AUDIENCE
+  clientId: process.env.CAMUNDA_CLIENT_ID,
+  clientSecret: process.env.CAMUNDA_CLIENT_SECRET,
+    // These settings come from your .env file. Note that CAMUNDA_TOKEN_AUDIENCE is represented by ZEEBE_TOKEN_AUDIENCE.
+  audience: process.env.CAMUNDA_TOKEN_AUDIENCE
 };
 
 // An action that lists all roles.
 async function listRoles() {
   // Every request needs an access token.
   const accessToken = await getAccessToken(authorizationConfiguration);
 
-  // These settings come from your .env file.
-  const camundaApiUrl = process.env.ZEEBE_BASE_URL;
+  // These settings come from your .env file. Note that CAMUNDA_REST_ADDRESS is represented by ZEEBE_REST_ADDRESS.
```

Comment:
I don't think this is accurate. `CAMUNDA_REST_ADDRESS` is represented by `CAMUNDA_REST_ADDRESS` (same name) in the .env.example file (which will likely turn into their .env file.)

It might be that `CAMUNDA_REST_ADDRESS` is represented by `ZEEBE_REST_ADDRESS` in the .env variables that you can download when creating a client....but that's not how I read this comment.

Reply:
Does my latest commit clarify this?

Reply:
Yes!